mobile/src/test/java/org/openhab/habdroid/util/UtilTest.java

```diff
@@ -42,16 +44,31 @@ public void normalizeUrl() throws Exception {
 
     @Test
     public void parseSitemapList() throws Exception {
-        List<OpenHABSitemap> sitemapList = Util.parseSitemapList(createSitemapDocument());
+        // openHAB 1
+        List<OpenHABSitemap> sitemapList = Util.parseSitemapList(createSitemapOH1Document());
         assertFalse(sitemapList.isEmpty());
 
         // Should be sorted, null first
         assertEquals(null, sitemapList.get(0).getLabel());
         assertEquals("Garden", sitemapList.get(1).getLabel());
         assertEquals(8, sitemapList.size());
+
+        // openHAB 2
```

Comment:
can you extract that as a separate unit tests? So, you can save the comment and it's easier to find the failing test, if you see what action was done. You also do not need to have multiple setup steps in that way :)